cmd/zoekt-sourcegraph-indexserver/debug.go

```diff
@@ -147,6 +154,98 @@ func debugListIndexed() *ffcli.Command {
 	}
 }
 
+func debugQueue() *ffcli.Command {
+	var connectionTimeout time.Duration
+	var printHeader bool
+
+	var serverAddress string
+
+	fs := flag.NewFlagSet("debug queue", flag.ExitOnError)
+	fs.DurationVar(&connectionTimeout, "timeout", 10*time.Second, "max timeout for establishing a connection to a zoekt-sourcegraph-indexserver instance")
```

Comment:
for the other debug commands we use fs.<Type> instead of fs.<Type>Var. Just the root flogs use the fs.<Type>Var because they set fields on the struct.